### DISCUSSION TITLE
Graphql Code Generator

### DISCUSSION THREAD

**Discussion by pennan88** [ChilliCream/hotchocolate#5337] [Q&A] [answered]
Hello everyone! Quick question.

I have been working on a Blazor WASM project and got the idea that I should try React/Asp.net and was thinking about the Graphql Code Generator I used a while back in a Next.JS project to get the types.

Is it even possible to have a ASP.net backend server with hot chocolate and use the [Graphql Code Generator](https://www.graphql-code-generator.com/) to get the type for a React frontend? I tried it real quick and ended up getting "Failed to load schema from "https://localhost:7181/graphql/"".

Thanks

**Comment by matt-psaltis** [comment, marked as answer]
We've used the same library on a recent project and it works. FIrst thing that comes to mind is the ssl cerificate. Make sure that's disabled during the code gen. We use cross-env NODE_TLS_REJECT_UNAUTHORIZED=0 on the generate step in our package.json for example.

**Reply by pennan88** [reply]
Yes! That was the problem, works fine now. Thanks alot 👍
